source/common/matcher/solo_matcher.cc

```diff
@@ -38,8 +38,8 @@ class BaseMatcherImpl : public Matcher,
 
     matches &= Http::HeaderUtility::matchHeaders(headers, config_headers_);
     if (!config_query_parameters_.empty()) {
-      Http::Utility::QueryParams query_parameters =
-          Http::Utility::parseQueryString(
+      auto query_parameters =
```

Comment:
is query_parameters no longer of type `Http::Utility::QueryParams`? Did the type name change?